x-pack/plugins/security_solution/public/cases/components/settings/jira/search_issues.tsx

```diff
@@ -0,0 +1,104 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React, { useMemo, useEffect, useCallback, useState, memo } from 'react';
+import { EuiComboBox, EuiComboBoxOptionOption } from '@elastic/eui';
+
+import { HttpSetup, ToastsApi } from 'kibana/public';
+import { ActionConnector } from '../../../containers/types';
+import { useGetIssues } from './use_get_issues';
+import { useGetSingleIssue } from './use_get_single_issue';
+import * as i18n from './translations';
+
+interface Props {
+  selectedValue: string | null;
+  http: HttpSetup;
+  toastNotifications: Pick<
+    ToastsApi,
+    'get$' | 'add' | 'remove' | 'addSuccess' | 'addWarning' | 'addDanger' | 'addError'
+  >;
+  actionConnector?: ActionConnector;
+  onChange: (parentIssueKey: string) => void;
+}
+
+const SearchIssuesComponent: React.FC<Props> = ({
+  selectedValue,
+  http,
```

Comment:
I think these props are coming from `useKibana` hook, we should just continue to use this hook instead of passing them through the props